Description: Fix '--inherit' option behavior
 Process options in reverse order for better templating.
 .
 (see http://lists.unbit.it/pipermail/uwsgi/2011-June/002182.html)
Origin: upstream, http://projects.unbit.it/hg/uwsgi/rev/5cccb38b8b70
Last-Update: 2011-06-12

Index: uwsgi/uwsgi.c
===================================================================
--- uwsgi.orig/uwsgi.c	2011-06-12 15:19:05.000000000 +0300
+++ uwsgi/uwsgi.c	2011-06-12 15:20:00.000000000 +0300
@@ -229,7 +229,7 @@
 	int is_retry;
 	int found;
 
-	for (i = 0; i < uwsgi.exported_opts_cnt; i++) {
+	for (i = uwsgi.exported_opts_cnt-1; i >= 0; i--) {
 
 		if (uwsgi.exported_opts[i]->configured)
 			continue;
